packages/FormElement/src/FormElement.styles.js

```diff
@@ -24,6 +25,7 @@ export const FormElement = styled.div(
     ${isInline && inlineFormElementStyles};
     ${isDisabled && `opacity: 0.5;`}
     border: none;
+    margin: 0 0 ${tokens.space} 0;
```

Comment:
now ensuring a bottom margin is specified only on the root element. This is important to ensure it won't be overridden by application css eg when it is rendered as a `fieldset`

Reply:
Could we alternatively set `margin: 0` and rely on the app to apply any bottom margin it needs?  

I think it's ideal for the root element of a component to have no margin.

Reply:
Sure, yeah that probably makes more sense. Will change.